### PR TITLE
Added recipe for a stackless meta package

### PR DESCRIPTION
**Pull request by akruis**
This pull request adds a recipe for a meta package, which tracks the feature "stackless". 

I created another pull request (https://github.com/conda-forge/python-feedstock/pull/131) to package Stackless Python as a "python" with the feature "stackless". Because Stackless Python is a complete Python and conflicts with regular C-Python the feature-mechanism is used to select it over regular C-Python.

### CODE REVIEW

**Comment by akruis** [comment]
Any chance to get the recipe linted or to disable the linter? The meta.yaml file is nearly completely empty, because it is recipe for a meta package.

**Comment by jakirkham** [comment]
cc @jjhelmus

File: recipes/stackless/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  string: ''
+  track_features:
+    - stackless
+
+requirements:
+  build: []
+  run: []
+
+test:
+  imports: []
```
Comment (scopatz) [inline recipes/stackless/meta.yaml]: This needs tests of some sort...

**Comment by jakirkham** [comment]
Perhaps we can build off the work that has been done for PyPy so far to enable Stackless. ( https://github.com/conda-forge/conda-forge.github.io/issues/867 ) WDYT @isuruf? 🙂

**Pull request closed without merging**